modules/gltf/gltf_document.cpp

```diff
@@ -79,6 +78,9 @@
 #include "modules/gridmap/grid_map.h"
 #endif // MODULE_GRIDMAP_ENABLED
 
+// FIXME: Hardcoded to avoid editor dependency.
+#define GLTF_IMPORT_USE_NAMED_SKIN_BINDS 16
```

Comment:
This needs further work from @godotengine/import. Either the enum can be replicated, or it should be moved to a non-editor file.

Reply:
See previous attempt https://github.com/godotengine/godot/pull/48914 Can ask @reduz if he approves?

Reply:
That's one way to solve it yeah :)